Apps/phdossier/dossier_connector.py

```diff
@@ -0,0 +1,342 @@
+# File: dossier_connector.py
+# Copyright (c) 2019 Splunk Inc.
+#
+# Licensed under Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0.txt)
+import phantom.app as phantom
+from phantom.base_connector import BaseConnector
+from phantom.action_result import ActionResult
+
+# Usage of the consts file is recommended
+# from dossier_consts import *
+import json
+import requests
+from requests.auth import HTTPBasicAuth
+# from bs4 import BeautifulSoup
+
+
+class RetVal(tuple):
+    def __new__(cls, val1, val2=None):
+        return tuple.__new__(RetVal, (val1, val2))
+
+
+class DossierConnector(BaseConnector):
+
+    def __init__(self):
+
+        # Call the BaseConnectors init first
+        super(DossierConnector, self).__init__()
+
+        self._state = None
+
+        # Variable to hold a base_url in case the app makes REST calls
+        # Do note that the app json defines the asset config, so please
+        # modify this as you deem fit.
+        self._base_url = None
+
+    def _make_rest_call(self, endpoint):
+
+        base_url = "https://api.activetrust.net:8000/api/"
```

Comment:
`services/intel/lookup/indicator` can be here and taken out of each of the action's urls

Reply:
Fixed.